.github/workflows/vacuum.yml

```diff
@@ -0,0 +1,19 @@
+name: Vacuum OpenAPI lint
```

Comment:
Did you consider integrating this functionality into the [Unified CI](https://github.com/camunda/camunda/wiki/CI-&-Automation#implementation) instead of creating a new dedicated workflow? I want to avoid a high number of small dedicated workflows if possible and those workflows also won't block anyone from merging, since they are no required status checks in GitHub.

Reply:
I actually did, tried it timeboxed for 30 minutes but go stuck on integrating it properly.
I decided to go for a separate check for now that is not required to make progress.
Happy to iterate on this moving forward and integrating this into main CI flow.

Reply:
Created a follow-up issue with https://github.com/camunda/camunda/issues/24267

Reply:
> I actually did, tried it timeboxed
Nice!

> happy to pair and learn how to integrate this
Let's do this then :) I'm back next week and will take a cursory look at https://github.com/camunda/camunda/issues/24267 then reach out